lib/get_parser.py

```diff
@@ -324,7 +324,23 @@ def get_parser() -> argparse.ArgumentParser:
         action="store_true",
         help="Use jobsubdevgpvm01 etc. to submit",
     )
-
+    parser.add_argument(
+        "--onsite",
+        dest="usage_model",
+        action="store_const",
+        const="OPPORTUNISTIC,DEDICATED",
+        default="OPPORTUNISTIC,DEDICATED,OFFSITE",
+        help="run jobs locally only; " "usage_model=OPPORTUNISTIC,DEDICATED",
+    )
+    parser.add_argument(
+        "--offsite",
+        dest="usage_model",
+        action="store_const",
+        const="OPPORTUNISTIC,DEDICATED,OFFSITE",
+        default="OPPORTUNISTIC,DEDICATED,OFFSITE",
+        help="run jobs offsite if slots slots are available; "
+        "usage_model=OPPORTUNISTIC,DEDICATED,OFFSITE",
+    )
```

Comment:
What do you think about making the two flags `--onsite` and `--offsite` mutually exclusive?  I know argparse has something like `mutually_exclusive_group` to do that if you think it's a good idea.

Reply:
This makes sense. I will look into it. I was holding the user to a higher level of responsibility that you are are!

Reply:
done